src/NServiceBus.RavenDB/Timeouts/QueryTimeouts.cs

```diff
@@ -80,8 +80,18 @@ public QueryTimeouts(IDocumentStore documentStore, string endpointName)
             return new TimeoutsChunk(results, nextTimeToRunQuery);
         }
 
+        internal void Shutdown()
+        {
+            abort = true;
```

Comment:
Would it be more inline with other cancellation examples when we used a `CancellationTokenSource` internally?